console/create_app/deps.ts

```diff
@@ -1,2 +1,2 @@
-export { Drash } from "https://deno.land/x/drash@v1.0.0/mod.ts";
-export { assertEquals } from "https://deno.land/std@v0.52.0/testing/asserts.ts";
+export { Drash } from "https://deno.land/x/drash@v1.1.1/mod.ts";
+export { assertEquals } from "https://deno.land/std@v0.61.0/testing/asserts.ts";
```

Comment:
The first test that failed is because we changed this, but the tests haven't been reflected, eg tests still look for v1.0.0 drash, and the other tests fail because when a test fails, it doesn't clean up any files/dirs made, which is why any test after says "... exists"

Reply:
so what's a solution so we don't run into this issue when we update this file?

Reply:
I guess there isn't much we could do, unless adding a `cleanUpEnv()` method or something (i think it already exists) at the top of **each test**, so when a test does fail, the next one(s) would still run

Reply:
alright, since this is a clean up PR, i may as well put that work into this PR

Reply:
b48c60d, 51dd482, 33345c4